app/views/notices/index.njk

```diff
@@ -0,0 +1,217 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from "govuk/components/pagination/macro.njk" import govukPagination %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block breadcrumbs %}
+  {{ govukBackLink({
+    text: 'Back',
+    href: backLink
+  }) }}
+{% endblock %}
+
+{% block content %}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: error.summary
+    }) }}
+  {% endif %}
+
+  {% set filtersForm %}
+    <form  method="post">
+      <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
+
+      {# Filter by licence issues #}
+      {{ govukCheckboxes({
+        classes: "govuk-checkboxes--small",
+        name: "filterNotificationTypes",
+        fieldset: {
+          legend: {
+            text: "Notification type",
+            isPageHeading: false
+          }
+        },
+        items: [
+          {
+            value: "legacy-notifications",
```

Comment:
I think this is missing filter by ad-hoc ?

Reply:
it's not on the list I have in the ticket but can double check with alan on standup